File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int check;
        for(int i = 0; i < nums.size(); i++){
            check = nums[i];
            for(int j = i+1; j < nums.size(); j++){
                if(check == nums[j]){
                    return true;
                }
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        end = len(nums) - 1
        print(end)
        index = 0
        while True:
            jump = nums[index]
            index += jump

            if (jump == 0) and (index != end):
                return False

            if (index == end):
                return True
            elif (index > end):
                return False
            

        